src/main/g8/$name__norm$-api/src/main/scala/$package$/api/$name__Camel$Service.scala

```diff
@@ -73,3 +73,20 @@ object GreetingMessage {
     */
   implicit val format: Format[GreetingMessage] = Json.format[GreetingMessage]
 }
+
+
+
+/**
+  * The greeting message class used by the topic stream.
+  * Different than [[GreetingMessage]], this message includes the name (id).
+  */
+case class GreetingMessageChanged(name: String, message: String)
```

Comment:
I choose to not have a sealed trait here, because otherwise we would have to introduce hand made formatters or  https://github.com/julienrf/play-json-derived-codecs which sounds a little bit too much for a simple hello world app.